api/v1beta1/awscluster_webhook_test.go

```diff
@@ -87,6 +87,20 @@ func TestAWSCluster_ValidateCreate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "HealthCheckProtocol in ControlPlaneLoadBalancer should default to ClassicELBProtocolSSL",
```

Comment:
As we have agreed on [this](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea) convention, so can we use it here ?

Reply:
Done